wordpress-mobile/WordPress-Android
# Update login library from subtree

Brings in the latest changes to the login library, imported from WooAndroid.

Changes:

* Two crash fixes: https://github.com/woocommerce/woocommerce-android/pull/413 and https://github.com/woocommerce/woocommerce-android/pull/747
* A renamed resource

To test:

A cursory login flow test - also confirm that the illustration on the 'magic link sent' screen is correct.

Update release notes:
